apps/multisig/src/util/strings.tsx

```diff
@@ -9,3 +9,10 @@ export const parseURL = (url: string) => {
     return false
   }
 }
+
+export const removeSurroundingCharacters = (input: string): string => {
+  // Regular expression to remove the first and last characters if they are not letters or numbers
+  // Sample input: '“Santos”'
+  // Sample output: 'Santos'
+  return input.replace(/^[^a-zA-Z0-9]+|[^a-zA-Z0-9]+$/g, '')
+}
```

Comment:
wrapping values with double quotes is pretty common in CSV, to preserve any commas in the value.

Using double quote here is a very specific case, perhaps we should just check whether the value starts and end with double quote, and if so, simply remove them, rather than excluding any other symbols?

Consider using csv-parse too

```ts
import { parse } from "csv-parse";
```

Reply:
Hey I went with excluding non alphanumeric values mainly because the issue is coming from Apple Numbers, and it converts `"` to `“` as an opening and `”` as a closing quote and `'` to `‘` and  `’` respectively. Don't know if you can notice the difference here in github comments markdown, but those are actually two different symbols.


I figure instead of creating a more complex regex to check for those specific symbols, we should simply remove all non alphanumeric characters since they are not accepted by these inputs anyway

Reply:
ohhhh ok that makes sense, yea those are definitely different symbols lol